### PR TITLE
[UI] Using same color for pending and unscheduled is misleading

### LINKED ISSUE #1906: [UI] Using same color for pending and unscheduled is misleading
Two different states should not be represented with the same color. I'm not good at color picking, but grey for unscheduled seems okay. Could we get another color for pending?

### PR DESCRIPTION
**Pull request by km-js**
Fixes #1906

Changes Applied:
Changed the color of Pending status from grey to amber which is a warning color.
I hope that makes sense if not, please suggest me something better.

### CODE REVIEW

File: ui/src/utils/labels.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export default {
   RUNNING: 'info',
-  PENDING: 'default',
+  PENDING: 'warning',
```
Comment (helfi92) [inline ui/src/utils/labels.js]: We use `warning` for exception already. Also, the filter card color should change as well:

Screen Shot 2019-11-06 at 1 37 10 PM.

Comment (km-js) [reply ui/src/utils/labels.js]: Yeah, but all others are also in use already. What do you suggest in that case?

Comment (helfi92) [reply ui/src/utils/labels.js]: That's a good question. We could probably just use a class to override the color.

`<StatusLabel className={...} />`

Comment (km-js) [reply ui/src/utils/labels.js]: Hm.. But Isn't `StatusLabel `is used in various components?

**Comment by km-js** [comment]
Hey @helfi92 , any reviews on this?

**Comment by km-js** [comment]
> 
> 
> Pending has a yellow background at the top but the status label shows it as grey. Let's fix that.
> 
> Screen Shot 2019-11-12 at 9 26 55 AM

Hey, Can you help a bit on how to do that?
 As pending is coming from the state here:

`<StatusLabel state={taskGroup.status.state} />`

**Comment by helfi92** [comment]
You should be able to add a className:

```jsx
{
  background: `... !important`,
}

<StatusLabel state={taskGroup.status.state} className={...} />
```

**Comment by km-js** [comment]
Won't this change every status's background color to yellow rather than just 'pending'???

**Comment by helfi92** [comment]
> Won't this change every status's background color to yellow rather than just 'pending'???

Yes. You'll need to add to conditionally add this class.

**Comment by km-js** [comment]
Hey @helfi92 , Are there any other changes required here?

**Pull request merged**